app/src/main/java/io/horizontalsystems/bankwallet/modules/releasenotes/ReleaseNotesViewModel.kt

```diff
@@ -0,0 +1,28 @@
+package io.horizontalsystems.bankwallet.modules.releasenotes
+
+import androidx.lifecycle.ViewModel
+import io.horizontalsystems.bankwallet.core.IAppConfigProvider
+import io.horizontalsystems.bankwallet.core.managers.ReleaseNotesManager
+import io.horizontalsystems.core.SingleLiveEvent
+
+class ReleaseNotesViewModel(
+        private val appConfigProvider: IAppConfigProvider,
+        releaseNotesManager: ReleaseNotesManager
+) : ViewModel() {
+
+    val releaseNotesUrl = releaseNotesManager.releaseNotesUrl
+    val openLinkLiveData = SingleLiveEvent<String>()
```

Comment:
What if the fragment will ask for url from viewModel for specific social network and open it synchronously?